### ISSUE TITLE
http://www.estadiofutebol.com/gratis/motorstv

### ISSUE THREAD

**Issue by Tronen** [reek/anti-adblock-killer#3030]
Filter needed for:
http://www.estadiofutebol.com/gratis/motorstv

This link is blocked in a mobile android phone with Adguard and these user filters:
https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer-filters.txt

**Comment by uBlock-user** [comment]
@X01X012013 same as https://github.com/reek/anti-adblock-killer/issues/3029

**Comment by jspenguin2017** [comment]
@uBlock-user 
This doesn't seem to be related, I think this rule from AAK List is breaking the website... `||cdn.tvdez.com^$third-party`

**Comment by jspenguin2017** [comment]
OK, this fixes it: 
```
@@||cdn.tvdez.com^$domain=www.estadiofutebol.com
||cdn.tvdez.com^$subdocument,domain=www.televisaofutebol.com
```

**Comment by uBlock-user** [comment]
@X01X012013 They're related, open the logger and you will see it's making a connection attempt to `televisaofutebol.com`

**Comment by jspenguin2017** [comment]
Do you have Flash installed?

**Comment by uBlock-user** [comment]
No, but the iframe should load even without flash, doesn't it ?

**Comment by jspenguin2017** [comment]
The whole frame is just a Flash player, it loads fine on my end.

**Comment by uBlock-user** [comment]
Fine, if it works for you, conclude as you may, I am leaving this one and https://github.com/reek/anti-adblock-killer/issues/3029 upto you as I don't have Flash installed.

**Comment by jspenguin2017** [comment]
@uBlock-user You are right, this IS related to `freetvall.com`. 
The `iframe`, `televisaofutebol.com` has some self-sabotage logic which cause it to send garbage based on cookies...

**Comment by uBlock-user** [comment]
@X01X012013 That wasn't happeing when I brought you in, 48 hours ago, so the webmaster is keeping tabs on this one.

**Comment by jspenguin2017** [comment]
@uBlock-user I don't know anymore, these websites redirect the user through a giant pile of garbage, and it's not the same pile of garbage every time, so it's hard to keep track of it.
